acceptance-tests.md

```diff
@@ -148,12 +175,14 @@ Reference for NERC OpenStack: [https://nerc-project.github.io/nerc-docs/get-star
     * _As an administrator of the cluster, I should be able to view daily, weekly and monthly reports of the cluster infrastructure utilization._
     * <span style="text-decoration:underline;">Steps</span>
         1. Administrator logs into the associated XDMoD instance and views reports.
+    * <span style="text-decoration:underline;">Acceptance tests:</span>
         2. [Click here to view the ACM Observability Grafana dashboards](https://multicloud-console.apps.nerc-ocp-infra.rc.fas.harvard.edu/grafana/). These dashboards provide insights into Control Plane Health, Optimization, Capacity, Utilization and more. You can change the timespan in the top right to show results in terms of minutes, hours, days, months or years.
 2. **Track/report usage of the project**
     * _As a user and the owner of a project, I should be able to view daily, weekly and monthly reports of the infrastructure utilization by the projects I own._
     * <span style="text-decoration:underline;">Steps</span>
         1. User logs into the associated XDMoD instance and views reports for projects they own.
         2. User cannot view reports for projects they do not own. We will need to look into this, to restrict the view to only projects that they own.
+    * <span style="text-decoration:underline;">Acceptance tests:</span>
         3. [Click here to view the memory usage of projects over time](https://multicloud-console.apps.nerc-ocp-infra.rc.fas.harvard.edu/grafana/explore?orgId=1&left=%5B%22now-1h%22,%22now%22,%22Observatorium%22,%7B%22exemplar%22:true,%22expr%22:%22namespace:container_memory_usage_bytes:sum%22%7D%5D).
         4. [Click here to view the cpu usage of the projects over time](https://multicloud-console.apps.nerc-ocp-infra.rc.fas.harvard.edu/grafana/explore?orgId=1&left=%5B%22now-1h%22,%22now%22,%22Observatorium%22,%7B%22exemplar%22:true,%22expr%22:%22node_namespace_pod_container:container_cpu_usage_seconds_total:sum%22%7D%5D).
-        5. [Click here to show the projects using the top 5 cpu usage at each point in time](https://multicloud-console.apps.nerc-ocp-infra.rc.fas.harvard.edu/grafana/explore?orgId=1&left=%5B%22now-1h%22,%22now%22,%22Observatorium%22,%7B%22exemplar%22:true,%22expr%22:%22topk(5,%20(1%20-%20avg(rate(node_namespace_pod_container:container_cpu_usage_seconds_total:sum%7B%7D%5B$__rate_interval%5D))%20by%20(namespace)))%22%7D%5D).
+        5. [Click here to show the projects using the top 5 CPU usage at each point in time](https://multicloud-console.apps.nerc-ocp-infra.rc.fas.harvard.edu/grafana/explore?orgId=1&left=%5B%22now-1h%22,%22now%22,%22Observatorium%22,%7B%22exemplar%22:true,%22expr%22:%22topk%285,%20%281%20-%20avg%28rate%28node_namespace_pod_container:container_cpu_usage_seconds_total:sum%7B%7D%5B$__rate_interval%5D%29%29%20by%20%28namespace%29%29%29%22%7D%5D).
```

Comment:
something about the new section broke the numbering please double-check that you have the expected carriage returns etc.

Reply:
Thanks for the great feedback here Kim, I have resolved the numbering issues.